src/tests/ftest/config_file_gen.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python3
 """
-  (C) Copyright 2020-2022 Intel Corporation.
+  (C) Copyright 2020-2023 Intel Corporation.
 
   SPDX-License-Identifier: BSD-2-Clause-Patent
 """
-
+# pylint: disable=fixme
```

Comment:
As I put above there is something odd about how this file is run and where it lives in the source tree, I think in order to run it it would either have to be copied to a different location or else run with a custom PYTHONPATH as the `from utli.*` import isn't working in the checker.  This means that some checks that pylint could run are not being run so we've not getting full coverage, but it's also a bad code smell that (I think) is indicating that our source code is laid out incorrectly.

I agree that we should move these annotations to the exact lines that require them and use disable-next to limit the scope of what's being disabled but I don't think that should block the PR.

There is an issue with this code however that's "odd" and we've already taken considerable time trying to unwind it and find out the root cause of the problem, the TODO could certainly be expanded to better explain our current understanding (vague such as it is) but to remove the marker noting that there are problems here would be a mistake.  From recollection there were 3-4 similar issues across the codebase when I worked on adding the pylint testing, the others I was able to resolve adequately but this one which affects not just this file but at least one more are still open questions.

Reply:
@ashleypittman Talked with Phil and we're okay to look into / fix this later